sdk/core/azure-core/src/main/java/com/azure/core/util/polling/PollerFlux.java

```diff
@@ -108,61 +187,12 @@ public void subscribe(CoreSubscriber<? super AsyncPollResponse<T, U>> actual) {
      */
     public SyncPoller<T, U> getSyncPoller() {
         return new DefaultSyncPoller<>(this.defaultPollInterval,
-                this.activationOperation,
+                this.syncActivationOperation,
```

Comment:
Here you can do `ctx -> this.activationOperation.apply(ctx).block()` instead of eagerly creating `syncActivationOperation`

Reply:
We could do it in previous version but not anymore since we don't have two class members defined to store variations of activationOperations

Reply:
@srnagar - or we could re-introduces those two member vars and use it for SyncPoller is created. What do you think?

Reply:
@anuchandy I guess re-introducing two variables will make it less readable. We can leave it as-is for now.

Reply:
Ok, sounds good.